src/modules/legacyReferrals/components/ProjectLegacyReferrals.tsx

```diff
@@ -0,0 +1,105 @@
+import AddIcon from '@mui/icons-material/Add';
+import { Stack } from '@mui/material';
+
+import ProjectOutgoingReferralPostingsTable from './ProjectOutgoingReferralPostingsTable';
+import ProjectReferralPostingsTable from './ProjectReferralPostingsTable';
+import ProjectReferralRequestsTable from './ProjectReferralRequestsTable';
+
+import ButtonLink from '@/components/elements/ButtonLink';
+import TitleCard from '@/components/elements/TitleCard';
+import { useGlobalFeatureFlags } from '@/hooks/useGlobalFeatureFlags';
+import { useProjectDashboardContext } from '@/modules/projects/components/ProjectDashboard';
+import { ProjectDashboardRoutes } from '@/routes/routes';
+import { generateSafePath } from '@/utils/pathEncoding';
+
+/**
+ * This component is used to show the legacy referrals page.
+ *
+ * If the project has CE referrals enabled, it disables the ability to create
+ * new legacy referrals and referral requests.
+ * TODO(#8142) fully sunset legacy referrals, remove this tab
+ */
+const ProjectLegacyReferrals = () => {
+  const { globalFeatureFlags: { externalReferralsEnabled } = {} } =
+    useGlobalFeatureFlags();
+  const { project } = useProjectDashboardContext();
+
+  // If this project has CE Referrals enabled (either Direct or Waitlist),
+  // show this page "legacy mode" which turns off the ability to create new
+  // legacy referrals and referral requests.
+  const { supportsReferrals, sendsDirectReferrals } =
+    project.coordinatedEntryFeatures || {};
+  const legacyMode = supportsReferrals || sendsDirectReferrals;
```

Comment:
This is minor but, here we are overloading the term "legacy" to mean two related but different things:
- Legacy Referrals = the old referrals (as opposed to CE referrals)
- Legacy Mode = a mode that the Legacy Referral UI can be in, only when CE is enabled

I think this overload might get confusing -- for example, for a client that is still using Legacy Referrals but does _not_ have CE enabled, we would say "they are using Legacy Referrals, but not in Legacy Mode."

Maybe we could rename `legacyMode` to something descriptive of what it does, like "disableNewLegacyReferrals"?

Reply:
good point, updated!